projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -577,17 +577,6 @@
 	}
 }
 
-// Overrides to make the preview look good
-.block-editor-inserter__preview {
-	.jetpack-contact-form {
-		padding: 16px;
-
-		.block-editor-inner-blocks .block-editor-block-list__layout {
-			margin: 0;
-		}
-	}
-}
-
```

Comment:
I'm not sure what this does, so I removed it. The preview styles look ok to me, and I couldn't see how this has any effect (even after fixing the classnames).

I did try checking the git blame, but the change isn't in the history of this repo. It looks like the code was moved across from someplace else at some point.

Reply:
Might be from times when preview was a small inline preview in sidebar, while now they are buttons with preview on hover.

Reply:
Comparing older commits to trunk and this branch I couldn't spot any spacing difference either. I think it's fine to remove. 

I did notice another recent minor regression around preview styles unrelated to this PR.